tests/test_modeling_vhead.py

```diff
@@ -0,0 +1,76 @@
+import unittest
```

Comment:
I think we can just call this the `test_modeling.py` file and add the `BaseModelTester`.

Reply:
Should be addressed now ;)